resources/Makefile.j2

```diff
@@ -198,4 +195,10 @@ pheval:
 	$(MAKE) pheval-run
 	$(MAKE) pheval-report
 
+.PHONY: all
+all:
+	$(MAKE) setup
+	$(MAKE) download-phenotype
```

Comment:
This does not make sense in the `resources/Makefile.j2`. Too specific to exomiser.